packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -75,6 +75,11 @@ export const messages = {
 
   force_source_deploy_text: 'SFDX: Deploy Source to Org',
   force_source_retrieve_text: 'SFDX: Retrieve Source from Org',
+  force_source_delete_text: 'SFDX: Delete from Project and Org',
+  force_source_delete_manifest_unsupported_message:
```

Comment:
@ruthemmanuelle Unlike the source retrieve and deploy commands, users cannot delete source by right-clicking on a manifest file because the `--manifest` option is not supported by the CLI. This is the message that will be displayed to the user if they do attempt to select the source delete command by right-clicking on a manifest file.
<img width="494" alt="screen shot 2018-08-28 at 1 47 16 pm" src="https://user-images.githubusercontent.com/1639375/44746900-f9725180-aac8-11e8-9e13-8ac4ace2dab5.png">

Reply:
Looks good, but the second sentence should end in a period.